### PR TITLE
Port System.Threading source comments to Docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Automatically ported some comments found in System.Threading.Channels and System.Threading.Tasks.Dataflow. Most of them are remarks.

### CODE REVIEW

**Comment by carlossanlop** [comment]
This PR is small.
Adding @rpetrusha @mairaw for language review.
Adding area owners @tarekgh @stephentoub for content review.

File: xml/System.Threading.Channels/ChannelReader`1.xml [hunk 1]
```diff
@@ -81,10 +81,18 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">The <see cref="T:System.Threading.CancellationToken" /> to use to cancel the enumeration.</param>
```
Comment (rpetrusha) [inline xml/System.Threading.Channels/ChannelReader`1.xml]:
```suggestion
        <param name="cancellationToken">The cancellation token to use to cancel the enumeration.</param>
```

File: xml/System.Threading.Channels/ChannelReader`1.xml [hunk 2]
```diff
@@ -81,10 +81,18 @@
...
+        <summary>Creates an <see cref="T:System.Collections.Generic.IAsyncEnumerable`1" /> that enables reading all of the data from the channel.</summary>
+        <returns>The created async enumerable.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+          
+Each <xref:M:System.Collections.Generic.IAsyncEnumerator`1.MoveNextAsync> call that returns <c>true</c> will read the next item out of the channel.
```
Comment (rpetrusha) [inline xml/System.Threading.Channels/ChannelReader`1.xml]:
```suggestion
Each <xref:System.Collections.Generic.IAsyncEnumerator%601.MoveNextAsync> call that returns `true` will read the next item out of the channel.
```

File: xml/System.Threading.Channels/ChannelReader`1.xml [hunk 3]
```diff
@@ -81,10 +81,18 @@
...
+
+<xref:M:System.Collections.Generic.IAsyncEnumerator`1.MoveNextAsync> will return `false` once no more data is or will ever be available to read.
```
Comment (rpetrusha) [inline xml/System.Threading.Channels/ChannelReader`1.xml]:
```suggestion
<xref:System.Collections.Generic.IAsyncEnumerator%601.MoveNextAsync> returns `false` once no more data is or will ever be available to read.
```

File: xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`2.xml [hunk 4]
```diff
@@ -305,7 +305,13 @@
       <Docs>
         <param name="exception">The <see cref="T:System.Exception" /> that caused the faulting.</param>
         <summary>Causes the <see cref="T:System.Threading.Tasks.Dataflow.IDataflowBlock" /> to complete in a  <see cref="F:System.Threading.Tasks.TaskStatus.Faulted" /> state.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+          
+After Fault has been called on a dataflow block, that block will complete (such that its <xref:M:Completion> task will enter a final state). Faulting a block, as with canceling a block, causes buffered messages (unprocessed input messages as well as unoffered output messages) to be lost.
```
Comment (rpetrusha) [inline xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`2.xml]:
```suggestion
After `Fault` has been called on a dataflow block, that block will complete, such that its <xref:System.Threading.Tasks.Dataflow.BatchedJoinBlock%602.Completion> task will enter a final state. Faulting a block, as with canceling a block, causes buffered messages (unprocessed input messages as well as unoffered output messages) to be lost.
```

File: xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`3.xml [hunk 5]
```diff
@@ -522,7 +522,13 @@
       <Docs>
         <summary>Returns a string that represents the formatted name of this  <see cref="T:System.Threading.Tasks.Dataflow.IDataflowBlock" /> instance.</summary>
         <returns>A string that represents the formatted name of this  <see cref="T:System.Threading.Tasks.Dataflow.IDataflowBlock" /> instance.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+          
+Inherited from <xref:T:System.Object>. Uses the <xref:P:System.Threading.Tasks.Dataflow.DataflowBlockOptions.NameFormat> option.
```
Comment (rpetrusha) [inline xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`3.xml]:
```suggestion
Inherited from <xref:System.Object>. Uses the <xref:System.Threading.Tasks.Dataflow.DataflowBlockOptions.NameFormat?displayProperty=nameWithType> option.
```

**Comment by rpetrusha** [comment]
Thanks, @carlossanlop. I'll merge now.

**Pull request merged**